.clang-format

```diff
@@ -0,0 +1,5 @@
+﻿---
+BasedOnStyle: Google
```

Comment:
Can we make the line length longer? I worry that the code will get pretty hard to read with the naming style we're using.

Reply:
I would agree with this. I think 100 is a good line length imo but 80 is quite short.

Reply:
I actually am a fan of 80column width, it's perfect to allow splitting an editor side by side without scrolling on my monitors. https://github.com/UWARG/ZeroPilot-SW-3/pull/10 has code that is restricted to 80 if you want to see a sample. I'm not the one working with the code the most though, so I don't think I get much say here

Reply:
Fair enough. I don’t work much with split editors so for me I’ve found a slightly longer line help, especially when dealing with function signatures that require a bunch of parameters and they’re verbose enough, 80 chars can cause lots of lines for a single function declaration. But tbh, I’m not too picky if there’s a clang format since it’ll deal with all the wrapping itself, not hard to deal with either way